### ISSUE TITLE
ConduitDataCollection doesn't include time and cycle number

### ISSUE THREAD

**Issue by greene30** [mfem/mfem#1841]
The ConduitDataCollection doesn't add the time and cycle number to the root file. They were both missing when I loaded the resulting root file into VisIt. I was able to fix this by adding the following lines to the SaveRootFile function inside the data collection:

```
  n_root["blueprint_index/mesh/state/time"] = time;
  n_root["blueprint_index/mesh/state/time_step"] = time_step;
  n_root["blueprint_index/mesh/state/cycle"] = cycle;
```

Thanks,
Patrick

**Comment by tzanio** [comment]
Thanks @greene30, do you want to propose a simple PR to fix this?

**Comment by cyrush** [comment]
let me check out something -- I think this might be fixed in conduit now (https://github.com/LLNL/conduit/pull/613), but I want to look at the MFEM useage.

**Comment by greene30** [comment]
I can setup the PR once @cyrush is done checking conduit (assuming it's still needed).

**Comment by cyrush** [comment]
yes, since we are using `blueprint::mesh::generate_index`, the fix in (https://github.com/LLNL/conduit/pull/613) should help here. @greene30 -- If possible - can you let me know the git sha of conduit that you are using? conduit::about() should provide a quick way to get it.

**Comment by cyrush** [comment]
also -- it won't hurt to make the change in mfem as well (it would just override the data) if you need to use an older version of conduit.

**Comment by greene30** [comment]
Here's the sha and version from conduit::about():

"version": "0.4.0",
"git_sha1": "63fb8f9889c468e2af630be7d2a3d2abb8947004"

This is the version ALE3D is currently using, but I noticed some one is in the process of updating to 0.5.1.

**Comment by cyrush** [comment]
thanks @greene30  -- pre 0.5.1 is already old -- It seems like making the change to add the details to the root file in mfem would help, b/c you may not be able to use the newer conduit in the near term.  Again, it won't hurt anything when used with newer conduit.

**Comment by greene30** [comment]
I'm looking through your CONTRIBUTING.md document right now. I'll try to get the PR setup some time this week.

**Comment by tzanio** [comment]
Thanks @greene30, let me know if you have any questions

**Comment by tzanio** [comment]
Fixed by #1864
